src/controllers/user.controller.ts

```
@@ -102,4 +108,58 @@
+    try {
+      const token = req.query.token as string;
+      if (!token) {
+        throw new InvalidTokenError('토큰이 필요합니다.');
+      }
+
+      const found = await this.userService.useToken(token);
+      if (!found) {
+        throw new TokenExpiredError();
+      }
+      
+      const { user, decryptedAccessToken, decryptedRefreshToken } = 
```

ESLint

[error] 'user' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)